### ISSUE TITLE
期待ARTrackV2

### ISSUE THREAD

**Issue by QingshuiL** [MIV-XJTU/ARTrack#71]
期待ARTrackV2的跟踪效果，请问预计什么时候能开源ARTrackV2的代码

**Comment by AlexDotHam** [comment]
We need to consult with the patent department on this point, and I hope that I can open-source my code as much as possible within my ability. If open source is not possible, I think you can easily reproduce our code because our structure is so simple that we only need a backbone and a mask decoder extracted from the MAE code.

Based on this codebase, you can make this in one day.

If u have any problem u can email me, or add me on WeChat if u want to, i will try my best to provide the file u need. #51

**Comment by AlexDotHam** [comment]
We release a temporal code for inference if you need that, the training code will be released when we finish the paper of ARTrackV3.
